src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  setFilterParamHandler: (param: string) => void
+};
+
+export const Filter: React.FC<Props> = ({ setFilterParamHandler }) => {
+  const [activeLink, setActiveLink] = useState({
```

Comment:
Instead of adding an extra state, you can pass `filterParam` to this component and use it to display active link